task-management/src/main/webapp/vue-app/tasks-management/components/tasks/TasksDashboard.vue

```diff
@@ -188,6 +188,11 @@
           window.setTimeout(() => this.tasks = this.tasks.filter((t) => t.id !== event.id), 500);
         }
       });
+      this.$root.$on('update-task-completed', (event) => {
+        if (event && !this.showCompleteTasks) {
+          window.setTimeout(() => this.tasks = this.tasks.filter((t) => t.id !== event.id), 500);
```

Comment:
Why you set a delay here?

Reply:
because there is a delay in all other similar event

Reply:
The delay was added to just wait a little and then the task disappear from the list (animation needs).